src/data/data.tsx

```diff
@@ -337,7 +337,7 @@ export const data: Array<{ name: string; data: techType }> = [
         {
           name: "Rest",
           link: "https://raw.githubusercontent.com/capcodigital/presentation-logos/master/tech-radar/integration/rest_square.png",
-          enabled: false,
+          enabled: true,
```

Comment:
The Rest is enabled but the content is not ready yet. Should we either disable it or otherwise prepare the content?

Reply:
Content is already there